source/app-sharing.rst

```diff
@@ -16,61 +18,67 @@ Overview
    home directory that other users can launch. This is similar to adding to your
    PATH the bin directory of another user.
 
-The rest of this document addresses OnDemand's support for Executable Sharing.
+File System Permissions
+-----------------------
 
-.. warning:: Executable sharing means the app and all its code runs as the user
-             executing it, like everything else in OnDemand. User's might not
-             realize this. We currently do not provide an opt in screen warning
-             users that this app "will have permission to do everything on their
-             behalf and act as them". As a result, you should fully trust whoever
-             you enable to do share apps using executable sharing.
+Admins may install apps to the system by copying the application directory to ``/var/www/ood/apps/sys``. Default directory permissions (``755``) will allow all users with access to Ondemand to see and run that app. Apps may have their access restricted by changing the permissions on individual application directories. For example if a site does not wish to show licensed software to un-licensed users they might do the following:
 
+  .. code-block:: sh
 
-Demo
-----
+    # Given:
+    # - an app named $NEW_APP
+    # - a group named $NEW_APP_GROUP
+    # - and a user named $NEW_APP_USER
 
-This is with two users Eric (efranz) and Bob (an0047).
+    sudo cp "/path/to/$NEW_APP" /var/www/ood/apps/sys
```

Comment:
Change this to

    sudo cp -r "/path/to/$NEW_APP" "/var/www/ood/apps/sys/$NEW_APP"